### PR TITLE
276 invalid regex phone

### LINKED ISSUE #276: [BUG] Invalid regex used in `phone->number`
## API Version
1.0
## Summary
The regex used [here](https://github.com/TOMP-WG/TOMP-API/blob/master/TOMP-API.yaml#L2211) is invalid. I assume it should be `^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\.0-9]*$`

Also, the example used [here](https://github.com/TOMP-WG/TOMP-API/blob/master/TOMP-API.yaml#L2212) is wrong. `examples` keyword should be used to give multiple examples of a phone number with each having distinct names.

I can create a PR that fixes this if you want.

### PR DESCRIPTION
**Pull request by edwinvandenbelt**
closes #276

### CODE REVIEW

File: TOMP-API.yaml [hunk 1]
```diff
@@ -2255,7 +2255,7 @@ components:
         number:
           description: phone number. In case of international usage, always provide the country code.
           type: string
-          pattern: '^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$'
+          pattern: '^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\.0-9]*$'
           example: +31-48934758 or +(0075)-834923384 or 020 1234 1234
```
Comment (canvural) [inline TOMP-API.yaml:2259]: This example is also wrong. If we want to show multiple examples we need to do
```yaml
examples:
    withCountryCodeAndDash:
        value: +31-48934758
    withCountryCodeAndParentheses:
        value: +(0075)-834923384
    withLeadingZero:
        value: 020 1234 1234
```

Comment (edwinvandenbelt) [reply TOMP-API.yaml:2259]: wow! I didn't knew this possibility (syntax). I've changed it (btw it is example: not examples:).

Comment (canvural) [reply TOMP-API.yaml:2259]: It is examples for multiple examples. https://swagger.io/docs/specification/adding-examples/

Comment (edwinvandenbelt) [reply TOMP-API.yaml:2259]: image
hmmm. The editor.swagger.io reports an error.

Comment (edwinvandenbelt) [reply TOMP-API.yaml:2259]: This seems a but in the editor.

Comment (edwinvandenbelt) [reply TOMP-API.yaml:2259]: O no, it's not a bug.. This 'examples' keyword is only allowed in the parameters part. Not in the object properties..

Comment (canvural) [reply TOMP-API.yaml:2259]: Oh, that's too bad :/

**Pull request merged**